automated-comments/go/raindrops/missing_entry_function.md

```diff
@@ -0,0 +1 @@
+The `Convert` function is missing or couldn't be found.
```

Comment:
What is the difference between missing and couldn't be found? Maybe just do:

```suggestion
The `Convert` function could not be found. Add the `Convert` function, which is required by the tests.
```

Reply:
Should this perhaps be a general comment, where the function name is passed as a parameter? That way, this comment can be re-used in multiple analyzers.